### PR TITLE
Update instructions to add Forking && screenshots of relevant Github UI

### PR DESCRIPTION
**Pull request by daviddeejjames**
🤞no weird formatting by my VScode
- (actually made me remove some old unused plugins from my home laptop)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -123,17 +131,37 @@ <h3>Step 4.</h3>
...
-      <b>Create a PR</b>
+      <h3>Step 9.</h3>
+      <p>Merge when you are happy with it, feel free to get a friend, mentor or
+        someone on your table to review it.</p>
 
-      <h3>Step 8.</h3>
-      <b>Merge when approved</b>
+      <h3>Step 10.</h3>
+      <p>Once you have merged your PR to your fork its time to get this live on
+        JuniorDev! To do this go to the <span class="mono">Pull Request</span> section in Github.
```
Comment (jamesformica) [inline index.html]: Do they go to their own Fork at this bit, or to the real HacktoberFEST repo? Just confirming

File: index.html [hunk 2]
```diff
@@ -123,17 +131,37 @@ <h3>Step 4.</h3>
...
+
+      <h3>Step 12.</h3>
+      <p>
+        Finally click the link that reads <span class="mono">Compare across forks</span>
+        <img class="doc-image" src='images/compare-forks.png' />
+      </p>
+
+      <h3>Step 13.</h3>
+      <p>Check it, submit it and then congrats you have sucessfully completed, not one but
+      <br><b>TWO</b> Pull Requests to <a href="https://hacktoberfest.digitalocean.com">#Hacktoberfest2019!</a></p>
```
Comment (jamesformica) [inline index.html:168]: 2️⃣ 🎉 👌 💯

**Review by jamesformica — APPROVED** [review]
👍 thanks for much for your help

File: index.html (hunk 1 above)
Comment (daviddeejjames) [reply index.html]: Ooo true its their own! I think you can do it from the real one too but for the sake of making it less confusing, Ill update the screenshot and copy stating own fork tyty

**Pull request merged**